getting_started/workflow/export/exporting_for_android.rst

```diff
@@ -9,24 +9,33 @@ The following steps detail what is needed to setup the SDK and the engine.
 Download the Android SDK
 ------------------------
 
-Download and install the Android SDK from
-`developer.android.com <https://developer.android.com/studio/>`__.
+Download and install the Android SDK. 
 
-If you install Android Studio, you need to run it once to complete the SDK setup.
+- You can install it using `Android Studio <https://developer.android.com/studio/>`__.
+  
+  - Run it once to complete the SDK setup using these `instructions <https://developer.android.com/studio/intro/update#sdk-manager>`__.
+  - Ensure that the `required packages <https://developer.android.com/studio/intro/update#recommended>`__ are installed as well.
```

Comment:
The link is to "Recommended packages", is that what you meant (instead of "required")?

Reply:
Under the `Recommended packages` section, there are a couple of `required` packages that need to be installed (the section title does make it weird to describe..).

Reply:
Ah right. I guess it's fine as is.